cmd/prometheus/main.go

```diff
@@ -164,6 +164,9 @@ func main() {
 	a.Flag("storage.remote.flush-deadline", "How long to wait flushing sample on shutdown or config reload.").
 		Default("1m").PlaceHolder("<duration>").SetValue(&cfg.RemoteFlushDeadline)
 
+	a.Flag("storage.remote.read-sample-limit", "Maxium number of samples to return via the remote read interface, in a single query.").
+		Default("20m").IntVar(&cfg.web.RemoteReadLimit)
```

Comment:
Default is `"20m"`, but the flag is an `IntVar` - running this results in `Error parsing commandline arguments: strconv.ParseFloat: parsing "20m": invalid syntax`

Reply:
So.. how to assess what max number of samples Prometheus can handle from remote read API?